ScheduleTask.BLL/Services/UserService.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using AutoMapper;
+using ScheduleTask.BLL.DTO;
+using ScheduleTask.BLL.Interfaces;
+using ScheduleTask.DAL.Entities;
+using ScheduleTask.DAL.Interfaces;
+
+
+namespace ScheduleTask.BLL.Services
+{
+    public class UserService : IUserService
+    {
+        IDataAccess Database { get; set; }
+
+        public UserService(IDataAccess db)
+        {
+            Database = db;
+        }
+
+        public void AddUser(UserDTO user)
+        {
+            User u = new User
```

Comment:
We should not use it this way. Saving data should be separately and using separate service